sdk/php/tests/SpecificationsDatum/SpecFilesTest.php

```diff
@@ -62,6 +62,25 @@ public function testFilesStructure()
             $root . '/sources/default/sources_list.json',
             $root . '/vehicles/default/marks.json',
             $root . '/vehicles/default/models.json',
+            $root . '/vehicles/default/models_agricultural.json',
+            $root . '/vehicles/default/models_artic.json',
+            $root . '/vehicles/default/models_atv.json',
+            $root . '/vehicles/default/models_autoloader.json',
+            $root . '/vehicles/default/models_bulldozer.json',
+            $root . '/vehicles/default/models_bus.json',
+            $root . '/vehicles/default/models_car.json',
+            $root . '/vehicles/default/models_construction.json',
+            $root . '/vehicles/default/models_crane.json',
+            $root . '/vehicles/default/models_dredge.json',
+            $root . '/vehicles/default/models_light_truck.json',
+            $root . '/vehicles/default/models_motorcycle.json',
+            $root . '/vehicles/default/models_municipal.json',
+            $root . '/vehicles/default/models_scooter.json',
+            $root . '/vehicles/default/models_self_loader.json',
+            $root . '/vehicles/default/models_snowmobile.json',
+            $root . '/vehicles/default/models_trailer.json',
+            $root . '/vehicles/default/models_truck.json',
+            $root . '/vehicles/default/vehicle_types.json',
```

Comment:
rename `vehicle_types.json` to `types.json`

Reply:
done